entrypoint

```diff
@@ -3,9 +3,6 @@ set -e
 BIND_ADDRESS="0.0.0.0"
 PUBLIC_PATH="public"
 LOG_DIR="."
-SERVER_URL="https://VelociraptorServer:8000/"
-DATASTORE_LOCATION="./"
```

Comment:
I see `DATASTORE_LCOATION` AND `FILESTORE_DIRECTORY` still referenced in line 15. Did you intend to remove these?